scripts/nrel_solar.json

```diff
@@ -0,0 +1,1363 @@
+{
+    "name" : "NREL-Solar",
+    "title" : "United States Solar Resources",
+    "description" : "These datasets detail the solar resource available in the United States",
+    "format" : "vector",
+    "spatial_ref" : "4326",
+    "citation" : "NREL",
+    "driver_name" : "ESRI Shapefile",
+    "keywords" : [
+        "solar",
+        "energy",
+        "spatial-data"
+    ],
+    "ref" : "https://www.nrel.gov/gis/data-solar.html",
+    "version" : "1.0.0",
+    "retriever" : "True",
+    "retriever_minimum_version" : "2.1.dev",
+    "extract_all" : "True",
+    "archived" : "zip",
+    "resources" : [
+        {
+            "name" : "us9805_dni",
```

Comment:
The name and the path don't match the contents in the path
I think it may be           
```
  "name" : "us9805_dni",
            "path" : "lower_48_dni_10km/us9805_dni.shp",
            "url" : "https://www.nrel.gov/gis/assets/data/lower_48_dni_10km.zip",
```

Reply:
`us9809_dni_updated.zip` is the updated archive to the older `lower_48_dni_10km.zip`